app/controllers/application_controller/tenancy.rb

```diff
@@ -9,11 +9,10 @@ module ApplicationController::Tenancy
   end
 
   def current_tenant
-    # current_user.try(:current_group).try(:tenant) || Tenant.default_tenant
     @current_tenant ||=
      #  Tenant.where(:subdomain => request.subdomains.last).first ||
      #  Tenant.where(:domain => request.domain).first ||
-      Tenant.default_tenant
+      current_user.try(:tenant) || Tenant.default_tenant
```

Comment:
> no, I like the way you did it

Meaning: I like the way it is currently implemented.

```
class User
  has_many :miq_groups
  has_many :tenants
  belongs_to :current_group
  belongs_to :current_tenant
end
```

Since we have many groups and many tenants, I like using `current_` there.
In the future, I'd like to get `current_` out of the `User`, but that is for "tomorrow".

DISCLAMER: code is aproximate not exact.